app/es_api/views.py

```diff
@@ -239,6 +240,49 @@ def get(self, request, doc_id):
             return HttpResponse(results, content_type="application/json")
 
 
+class GetSimilarCoursesView(APIView):
+    """This method defines an API for fetching results using the
+            more_like_this feature from elasticsearch for similar courses. """
```

Comment:
@mdonlon this explicitly does not use more_like_this, it uses a multi_match

Reply:
I forgot to fix this comment, will edit and push fix.

Reply:
still missing the update to this